IntercomDotnet/Resources/Users.cs

```diff
@@ -47,7 +47,7 @@ public dynamic DeleteWithUserId(string userId)
         [Obsolete("userId is a string in the Intercom API so please use the string-based overload going forward.")]
         public dynamic Get(string email = null, int? userId = null)
         {
-            return Get(email, userId?.ToString());
+            return Get(email, userId.ToString());
```

Comment:
What version of Visual Studio are you using? That syntax doesn't cause problems in the latest version of the C# language